scripts/hotspot.js

```diff
@@ -13,20 +13,25 @@
    * @param  {number} id
    * @param  {boolean} isSmallDeviceCB
    * @param  {H5P.ImageHotspots} parent
+   * @param  {string} icon
+   * @param  {image} iconImage
    */
-  ImageHotspots.Hotspot = function (config, color, id, isSmallDeviceCB, parent) {
+  ImageHotspots.Hotspot = function (config, color, id, isSmallDeviceCB, parent, icon, iconImage) {
     var self = this;
     this.config = config;
     this.visible = false;
     this.id = id;
     this.isSmallDeviceCB = isSmallDeviceCB;
+    // A utility variable so you don't have to always check if there is an iconImage.
+    var iconImageExists = (iconImage == null ? false : true); 
 
     if (this.config.content === undefined  || this.config.content.length === 0) {
       throw new Error('Missing content configuration for hotspot. Please fix in editor.');
     }
-
-    this.$element = $('<div/>', {
-      'class': 'h5p-image-hotspot',
+    
+    // Check if there is an iconImage that should be used instead of fontawesome icons to determine the html element.
+    this.$element = $((iconImageExists ? '<img/>' : '<div/>'), {
+      'class': 'h5p-image-hotspot ' + 'h5p-image-hotspot-' + icon,
```

Comment:
I would prefer:
1) not setting the icon-specific-class when an image is in use, like this:
`'class': 'h5p-image-hotspot ' + (!iconImageExists ? 'h5p-image-hotspot-' + icon : ''),`
2) Setting the src here, and not in the if below:
`src: iconImageExists ? H5P.getPath(iconImage.path, this.id) : undefined,`